documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,294 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+You can use {the-lc} {project-first} to migrate virtual machines from the following source providers to {virt} destination providers:
+
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
+* Open Virtual Appliances (OVAs) that were created by VMware vSphere
+* Remote {virt} clusters
+
+The release notes describe technical changes, new features and enhancements, known issues, and resolved issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Improved migration plan in the console
+
+The user interface console has improved the process of creating a migration plan.
```

Comment:
BTW, is there some more info on why it's a benefit to the user? Like, for example, what exactly improved in the process of creating a plan compared to the previous process?